core/internal/filetransfer/file_transfer_default.go

```diff
@@ -1,14 +1,21 @@
 package filetransfer
 
 import (
+	"context"
+	"encoding/base64"
 	"fmt"
 	"io"
 	"math"
 	"net/http"
 	"os"
 	"path"
 	"strings"
+	"time"
 
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
```

Comment:
I'm not sure why we're modifying the base `file_transfer_default.go` file to have Azure-specific functionality, while there is already a `file_transfer_azure.go` file? We have similar convention for other cloud providers.